docs/docusaurus/static/_redirects

```diff
@@ -369,4 +369,16 @@ docs/guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_azure
 /docs/guides/connecting_to_your_data/fluent/data_assets/how_to_organize_batches_in_a_sql_based_data_asset /docs/guides/connecting_to_your_data/fluent/database/sql_data_assets
 
 # Redirects for pages removed as no longer applicable
-/docs/guides/validation/advanced/how_to_validate_data_without_a_checkpoint /docs/guides/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint
\ No newline at end of file
+/docs/guides/validation/advanced/how_to_validate_data_without_a_checkpoint /docs/guides/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint
+
+# Redirects for removed index pages and new landing pages
+
+/docs/guides/setup/index /docs/guides/setup/setup_overview_lp
+/docs/guides/connecting_to_your_data/index /docs/guides/connecting_to_your_data/connect_to_data_lp
+/docs/guides/expectations/index /docs/guides/expectations/expectations_lp
+/docs/guides/validation/index /docs/guides/validation/validate_data_lp
+/docs/deployment_patterns/index /docs/category/integrations
+/docs/guides/validation/validate_data_overview /docs/guides/validation/validate_data_lp
+/docs/guides/expectations/create_expectations_overview /docs/guides/expectations/expectations_lp
+/docs/guides/connecting_to_your_data/connect_to_data_overview /docs/guides/connecting_to_your_data/connect_to_data_lp
+/docs/guides/setup/setup_overview /docs/guides/setup/setup_overview_lp
```

Comment:
This page also still exists and is in our ToC.  (Now as the "GX installation and configuration workflow".) This redirect is nonfunctional and can be deleted.  If we do want to redirect the old overview links to the landing pages, you'll need to update them manually instead (though I imagine that was already part of the other edits).